build-gluster-org/scripts/user-update.py

```diff
@@ -0,0 +1,99 @@
+import requests
+import re
+import os
+from git import Repo
+import subprocess
+
+#Collects the gluster-maintainers names
+def create_user_data(repo_path):
+    os.chdir(repo_path)
+    username = "rakshithakamath94"
```

Comment:
We do not use that variable, I think it can be removed

Reply:
I was basically trying to get the logic sorted out and I have updated everything and sent a new PR where I have removed all the usernames and stuff.